docs/reference/supervisor.md

```diff
@@ -2198,6 +2222,41 @@ The `wb_supervisor_simulation_reset` function sends a request to the simulator p
 The reset process is explained in detail in the [User Guide](https://www.cyberbotics.com/doc/guide/the-user-interface#file-menu), the only difference is that the supervisor and robot controllers are not restarted, if needed, they have to be restarted with the `wb_supervisor_node_restart_controller` function.
 You may wish to save some data in a file from your supervisor and robot controller programs in order to reload it when they restart.
 
+##### Reset/Reload Matrix
+
+The following table summarizes the behavior of different reset functions:
+- **Reload**: The reload button in the user-interace and the `wb_supervisor_world_reload` funtion.
+- **Reset from user interface**: The reset button in the user-interace.
+- **Reset from [Supervisor](supervisor.md)**: The `wb_supervisor_simulation_reset` function.
+- **Load node's state from Supervisor**: The `wb_supervisor_node_save/load_state` functions.
+- **Reset physics**: The `wb_supervisor_simulation_reset_physics` function.
+- **Reset node's physics**: The `wb_supervisor_node_reset_physics` function.
+
+| - | **Reload** | **Reset from user interface** | **Reset from [Supervisor](supervisor.md)** | **Load node's state from Supervisor** | **Reset physics** | **Reset node's physics** |
+|-|-|-|-|-|-|-|
+| **Resets simulation time** | Yes | Yes | Yes | No | No | No |
+| **Removes nodes** | Yes | Yes | Yes | No | No | No |
+| **Restarts controller** | Yes | Yes | No | No | No | No |
+| **Stops sounds** | Yes | Yes | Yes | No | No | No |
+| **Resets seeds** | Yes | Yes | Yes | No | No | No |
+| **Resets physics** | Yes | Yes | Yes | Scoped | Yes | Scoped |
+| **Resets physics plugin** | Yes | Yes | Yes | No | No | No |
+| **Resets all fields** | Yes | No | No | No | No | No |
+| **Adds removed nodes** | Yes | No | No | No | No | No |
+| **[Brake](brake.md)** | Resets | Releases | Releases | Releases | Nothing | Nothing |
```

Comment:
Instead of "Nothing" I would use "N/A" or "-" as "Nothing" let me think that it doesn't affect the node at all, but in reality it affects the physics of the node (but we already said it before). Ideally we should said "It doesn't change anything but the physics for this node", so maybe "N/A" (not applicable) or simply "-" means that we don't apply anything special in case of this node.

Reply:
`N/A` is good